src/libANGLE/renderer/metal/mtl_format_utils.h

```diff
@@ -81,6 +82,7 @@ struct Format : public FormatBase
     bool needConversion(angle::FormatID srcFormatId) const;
 
     MTLPixelFormat metalFormat = MTLPixelFormatInvalid;
+    CGColorSpaceRef metalColorspace = nil;
```

Comment:
I think only `CAMetalLayer` supports color space. Ordinary metal texture `MTLTexture` doesn't support it, hence adding metalColorspace to `Format` class is unnecessary. `Format` is currently used by both `CAMetalLayer` and textures.

Reply:
Okay, thanks for clarifying the difference. I will remove this in favor of `MGLDrawableColorFormat*` approach.